secret.js

```diff
@@ -0,0 +1,3 @@
+const JWT_SECRET = "ayush_Token";
```

Comment:
Better to create an environment and then put all secret keys into it. And make sure to add .env to .gitignore so that your secret won't get pushed to github.